### ISSUE TITLE
"Can the 'move' method only be used for relative movement? Can it be used to write game scripts?"

### ISSUE THREAD

**Issue by weizigege** [moses-palmer/pynput#619]
**Description**

**Platform and pynput version**
Your operating system and version, and the version of *pynput*.

**To Reproduce**
If possible, please include a short standalone code sample that reproduces the bug. Remember to surround it with [code block markers](https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks) to maintain indentation!

**Comment by moses-palmer** [comment]
Thank you for your report.

The current implementation of `move` does not actually provide relative movement on any platform. Rather, it reads the current cursor position and then modifies it, thus exhibiting a race condition where any movement from other sources between reading the position and setting the new one is discarded. I added the method with the intention of providing platform dependent implementations in the future however.

So in short, no, `move` in its current shape is not a good candidate for a game script. In particular, when the cursor is close to the screen edges, movement may be limited.
